.pipelines/.vsts-vhd-builder-release.yaml

```diff
@@ -110,6 +110,10 @@ parameters:
   displayName: Build 2204 ARM64 Gen2 containerd
   type: boolean
   default: true
+- name: build2404arm64gen2containerd
```

Comment:
no gen1?

Reply:
nope, I think we only needed to support Gen2 from the start. CC: @chengliangli0918

Reply:
arm64 vm size is new, so it has no gen1 only sku. Canonical also doesn't publish arm64 gen1 image in market place.